fuzz_lightyear/fuzzer.py

```diff
@@ -17,16 +17,18 @@
 
 def fuzz_parameters(
     parameters: List[Tuple[str, Dict[str, Any]]],
+    operation_id: str,
```

Comment:
What are your thoughts about making this optional? Like, if its provided, we can check the dictionary. Otherwise, we'll just fallback to the `.default_factory()`.